src/testRunner/unittests/tsbuildWatch/programUpdates.ts

```diff
@@ -796,4 +796,40 @@ createSomeObject().message;`,
             }),
         commandLineArgs: ["--b", "src/project", "-i", "-w"],
     });
+
+    verifyTscWatch({

```

Comment:
I think `tsc --watch` test is good enough ? we can remove this one.

Reply:
Sure, I'll delete it.